airflow/include/tasks/split.py

```diff
@@ -68,3 +69,45 @@ def split_python(dfs: list[pd.DataFrame]) -> pd.DataFrame:
     df.reset_index(inplace=True, drop=True)
 
     return df
+
+
+def split_html(dfs: list[pd.DataFrame]) -> pd.DataFrame:
+    """
+    This task concatenates multiple dataframes from upstream dynamic tasks and splits html code before importing
+    to a vector database.
+
+    param dfs: A list of dataframes from downstream dynamic tasks
+    type dfs: list[pd.DataFrame]
+
+    Returned dataframe fields are:
+    'docSource': ie. 'astro', 'learn', 'docs', etc.
+    'sha': the github sha for the document
+    'docLink': URL for the specific document in github.
+    'content': Chunked content in markdown format.
+
+    """
+
+    headers_to_split_on = [
+        ("h2", "h2"),
+    ]
+
+    df = pd.concat(dfs, axis=0, ignore_index=True)
+
+    splitter = HTMLHeaderTextSplitter(headers_to_split_on)
+
+    df["doc_chunks"] = df["content"].apply(lambda x: splitter.split_text(text=x))
+    df = df.explode("doc_chunks", ignore_index=True)
+    df["content"] = df["doc_chunks"].apply(lambda x: x.page_content)
+
+    # import tiktoken
```

Comment:
Why is this commented on? Maybe we should remove this.

Reply:
Good catch.  Removed in https://github.com/astronomer/ask-astro/pull/112/commits/41aa2f90920cc0983592586dd6666e854136501e